cmd/sealer/cmd/alpha/search.go

```diff
@@ -70,10 +71,15 @@ func NewSearchCmd() *cobra.Command {
 				}
 				tags, err := repo.Tags(context.Background()).All(context.Background())
 				if err != nil {
-					return err
+					return fmt.Errorf("repository %s does not exist, err:%v", imgName, err)
```

Comment:
errors.Wrapf(err,"repository %s does not exist", imgName)
 
 maybe is better here